views/HtmlJs.html

```diff
@@ -1,6 +1,8 @@
 <div class="s-q-item q-map" data-track-id="q-map" data-track-component-id="{{ id }}" style="opacity: 0">
   {{#if options.hideTitle !== true && hideTitle !== true }}
   <h3 class="s-q-item__title">{{title}}</h3>
+  {{/if}} {{#if (subtitle && subtitle !== '') || (subtitleSuffix && subtitleSuffix !== '') }}
```

Comment:
The `subtitleSuffix` is not needed here. I guess this is from Q-chart where the subtitleSuffix is e.g. `(in Millionen)`.

Reply:
I see. Removed in https://github.com/nzzdev/Q-map/pull/63/commits/b893653639ab78fe88a54a6442fc99e024fe28f4